src/engine/bufferscalers/enginebufferscalerubberband.cpp

```diff
@@ -54,13 +70,16 @@ void EngineBufferScaleRubberBand::setScaleParameters(double base_rate,
     // References:
     // https://bugs.launchpad.net/ubuntu/+bug/1263233
     // https://bitbucket.org/breakfastquay/rubberband/issue/4/sigfpe-zero-division-with-high-time-ratios
-    constexpr double kMinSeekSpeed = 1.0 / 128.0;
-    double speed_abs = fabs(*pTempoRatio);
-    if (speed_abs < kMinSeekSpeed) {
-        // Let the caller know we ignored their speed.
-        speed_abs = *pTempoRatio = 0;
-    }
+    // https://todo.sr.ht/~breakfastquay/rubberband/5
```

Comment:
the bitbucket link above is dead and can be removed

Reply:
Didn't want to remove it for legacy sake, but yes, thats why I added the sourcehut link.